src/patternfly.js

```diff
@@ -0,0 +1,17 @@
+/** PF Alert Component **/
+require('./pf-alert/pf-alert.component.js');
```

Comment:
Relative paths should work fine here and downstream (although I realize we had a different goals originally with separate repos). This is common in the Onsen UI project and the Angular2/React projects. Ex:
https://github.com/OnsenUI/OnsenUI/blob/master/core/src/elements/ons-alert-dialog/index.js#L18